### PR TITLE
Feature/add to collection

### PR DESCRIPTION
**Pull request by rej156**

### CODE REVIEW

File: packages/kauri-components/components/Card/ArticleCard.tsx [hunk 1]
```diff
@@ -385,13 +385,15 @@ interface IProps {
...
   ) => React.ReactElement<any>;
-  hoverChildren?: (
-    payload: {
-      hideDispatch: () => void;
-      showDispatch: () => void;
-      toggleDispatch: () => void;
-    }
-  ) => React.ReactElement<any>;
+  hoverChildren?:
+    | null
```
Comment (davodesign84) [inline packages/kauri-components/components/Card/ArticleCard.tsx]: You do miss ReasonML uh? :P

File: packages/kauri-components/components/Outline/index.tsx [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { Label } from "../Typography";
 import OutlineHeading from "./OutlineHeading";
 import styled from "../../lib/styled-components";
 import UserAvatarComponent from "../UserAvatar";
-import NFTList, { INFT } from '../Kudos/NFTList';
+import NFTList from "../Kudos/NFTList";
```
Comment (davodesign84) [inline packages/kauri-components/components/Outline/index.tsx]: Does the INFT get imported too? or you need to import * as ?

File: packages/kauri-components/components/Card/ArticleCard.tsx (hunk 1 above)
Comment (rej156) [reply packages/kauri-components/components/Card/ArticleCard.tsx]: https://twitter.com/bobzhang1988/status/1081773828341002241
https://codeburst.io/inference-engines-5-examples-with-typescript-flow-and-reason-edef2f4cf2d3

Yeah genuinely do

File: packages/kauri-components/components/Outline/index.tsx (hunk 2 above)
Comment (rej156) [reply packages/kauri-components/components/Outline/index.tsx]: It does get imported
Install tslint + prettier in your editor pl0x

**Pull request merged**